lib/rucio/tests/test_filter_engine.py

```diff
@@ -166,8 +186,43 @@ def test_OperatorsEqualNotEqual(self, session=None):
         dids = set(dids)
         self.assertEqual(list(map(lambda did: did.name in (did_name1, did_name2, did_name3), dids)).count(True), 2)     # 1, 3 (NULL counted in not equals)
 
+        # Plugin: JSON
+        #
+        if json_implemented(session=session):
+            did_name1 = self._create_tmp_DID()
+            did_name2 = self._create_tmp_DID()
+            did_name3 = self._create_tmp_DID()
+            set_metadata(scope=self.tmp_scope, name=did_name1, key='testkeyint1', value=1)
+            set_metadata(scope=self.tmp_scope, name=did_name2, key='testkeyint2', value=2)
+            set_metadata(scope=self.tmp_scope, name=did_name3, key='testkeyint3', value=2)
+
+            dids = []
+            q = FilterEngine('testkeyint1=1', model_class=models.DidMeta, strict_coerce=False).create_sqla_query(
+                additional_model_attributes=[
+                    models.DidMeta.scope,
+                    models.DidMeta.name
+                ],
+                json_column=models.DidMeta.meta)
+            dids += [did for did in q.yield_per(5)]
+            dids = set(dids)
+            self.assertEqual(list(map(lambda did: did.name in (did_name1, did_name2, did_name3), dids)).count(True), 1)
```

Comment:
equality supported by all sqla flavours, rest of the cases are not supported by oracle yet